src/main/java/course_project/Field.java

```diff
@@ -0,0 +1,299 @@
+package course_project;
+
+import lombok.Data;
+import lombok.Value;
+
+import java.util.*;
+
+@Data
+public class Field {
```

Comment:
Good! The game is nice to play, the interface is good, playing is easy. Approved!

Reply:
Thanks for the science and advices!